docs/upgrade-to-v5.md

```diff
@@ -100,3 +100,17 @@ The following deprecated methods have been removed:
 - `ObjectId.prototype.get_inc`
 - `ObjectId.get_inc`
   - The `static getInc()` is private since invoking it increments the next `ObjectId` index, so invoking would impact the creation of subsequent ObjectIds.
+
+### Negative Zero is now serialized to Double
+
+BSON serialize will now preserve negative zero values as a floating point number.
+
+Previously it was required to use the `Double` type class to preserve `-0`:
+```ts
+BSON.serialize({ d: new Double(-0) })
+```
+
+Now `-0` can be used directly
+```ts
+BSON.serialize({ d: -0 })
+```
```

Comment:
Maybe we should highlight the difference in behavior here?
```suggestion
Previously it was required to use the `Double` type class to preserve `-0`:
```ts
BSON.deserialize(BSON.serialize({ d: new Double(-0) })); // { d: Double(-0) }
BSON.deserialize(BSON.serialize({ d: new -0})); // { d: 0 }
```

Now `-0` can be used directly
```ts
BSON.deserialize(BSON.serialize({ d: new -0})); // { d: Double(-0) }
```
```

Reply:
The syntax here is a little broken, did you intend `new -0`? and deserialize does not return a Double instance unless you disable `promoteValues`. There were no changes to deserialize because it will return a negative zero js number if one is encoded in the BSON bytes so there's nothing to showcase with deserialize here

Reply:
updated